### ISSUE TITLE
name of inventory

### ISSUE THREAD

**Issue by noy98** [SkriptLang/Skript#1959]
### Description

I try to recognize the name of inventory, but it is weird.
It says "container.crafting"
### Steps to Reproduce

`on inventory close:
	broadcast "%player% - %uncolored name of player's current inventory%"
	if uncolored inventory name of current inventory of player contains "GUI설정":
		set {_splited::*} to uncolored inventory name of current inventory of player split at "-"
		set {_name} to {_splited::2}
		delete {gui.item.%{_name}%::*}
		set {_loop} to 9*{gui.line.%{_name}%}
		loop {_loop} times:
			add 1 to {_INT.%player%}
			set {gui.item.%{_name}%::%{_INT.%player%}%} to slot {_INT.%player%}-1 of current inventory of player
		send "{@p} &e&l%{_name}% &f&lGUI가 설정 되었습니다."`

### Errors / Screenshots

![2019-03-17_14 17 48](https://user-images.githubusercontent.com/27529365/54485653-9fe43f80-48bf-11e9-8fef-04af7378a6f9.png)
![2019-03-17_14 17 51](https://user-images.githubusercontent.com/27529365/54485655-a2469980-48bf-11e9-9706-df21621b0e44.png)

### Server Information
* **Server version/platform:**       1.12.2 
* **Skript version:**                2.3.6

**Comment by Blueyescat** [comment]
Why open a new issue... https://github.com/SkriptLang/Skript/issues/1957

**Comment by FranKusmiruk** [comment]
Labeling this as enhancement since it is the expected behaviour but not the desired one, I'd say this should return null when no inventory is open.

**Comment by sovdeeth** [comment]
I don't think this is undesired behavior. The server does not distinguish between "no inventory open" and "player's own inventory open", so in both cases it returns "Crafting", since the player's crafting inventory is their current inventory.
